pkgs/development/python-modules/skorch/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pytest
+, pytestcov
+, flaky
+, numpy
+, pytorch
+, scikitlearn
+, scipy
+, tabulate
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "skorch";
+  version = "0.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1wy4n842fm7jrccxnxh1a3psdm9d7k69vrsdb595246vpmlm6gv4";
+  };
+
+  propagatedBuildInputs = [ numpy pytorch scikitlearn scipy tabulate tqdm ];
+  checkInputs = [ pytest pytestcov flaky ];
+
+  checkPhase = ''
+    runHook preCheck
+    pytest -k 'not test_load_cuda_params_to_cpu and not test_pickle_load'
+    runHook postCheck
+  '';
+  # on CPU, these expect artifacts from previous GPU run
+
+  meta = with stdenv.lib; {
+    description = "Scikit-learn compatible neural net library using Pytorch";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ bcdarwin ];
```

Comment:
please add homepage, otherwise LGTM

Reply:
Done, somehow lost that commit.  Thanks!